cypress/pageObjects/datasetsPage.ts

```diff
@@ -0,0 +1,7 @@
+class DatasetPage {
```

Comment:
You should extend BasePage here and add elements to the page and init them in constructor.